spec.emu

```diff
@@ -669,27 +1596,22 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
           <emu-alg>
             1. For each ExportEntry Record _e_ of _module_.[[IndirectExportEntries]], do
               1. Let _resolution_ be _module_.ResolveExport(_e_.[[ExportName]]).
-              1. If _resolution_ is *null* or ~ambiguous~, throw a *SyntaxError* exception.
+              1. If _resolution_ is either *null* or ~ambiguous~, throw a *SyntaxError* exception.
               1. Assert: _resolution_ is a ResolvedBinding Record.
             1. Assert: All named exports from _module_ are resolvable.
             1. Let _realm_ be _module_.[[Realm]].
             1. Assert: _realm_ is not *undefined*.
             1. Let _env_ be NewModuleEnvironment(_realm_.[[GlobalEnv]]).
             1. Set _module_.[[Environment]] to _env_.
             1. For each ImportEntry Record _in_ of _module_.[[ImportEntries]], do
-              1. Let _importedModule_ be GetImportedModule(_module_, _in_.[[ModuleRequest]]).
+              1. Let _importedModule_ be GetImportedModule(_module_, _in_.[[ModuleRequest]]<ins>.[[Specifier]]</ins>).
               1. If _in_.[[ImportName]] is ~namespace-object~, then
                 1. Let _namespace_ be GetModuleNamespace(_importedModule_).
                 1. Perform ! _env_.CreateImmutableBinding(_in_.[[LocalName]], *true*).
                 1. Perform ! _env_.InitializeBinding(_in_.[[LocalName]], _namespace_).
-              1. <ins>Else if _in_.[[ImportName]] is ~module~, then</ins>
-                1. <ins>Let _moduleSourceObject_ be _importedModule_.[[ModuleSourceObject]].</ins>
-                1. <ins>If _moduleSourceObject_ is *null*, throw a *TypeError* exception.</ins>
-                1. <ins>Perform ! _env_.CreateImmutableBinding(_in_.[[LocalName]], *true*).</ins>
-                1. <ins>Perform ! _env_.InitializeBinding(_in_.[[LocalName]], _moduleSourceObject_).</ins>
```

Comment:
I just realized that my review above (https://github.com/tc39/proposal-import-reflection/pull/36#pullrequestreview-1410694415) is exactly what has been deleted here. What is the reason for deleting it?